mockups/citysourced/citysourced-reporter-panels-ng.html

```diff
@@ -60,11 +60,11 @@ <h2>Your Report Was Not Submitted</h2>
 
 		<div class="bc-citysourced-reporter" ng-show="!reporter.isDone">
 
-			<ol class="bc-citysourced-reporter-steps">
+			<ol class="bc-citysourced-reporter-steps" >
 				<li class="highlight">Choose a <br/>Report Type</li>
 				<li ng-class="{'highlight': reporter.page > 1}"
-					ng-show="reporter.shouldRequireLocation()">Enter a <br/>Location</li>
-				<li ng-class="{'highlight': reporter.page > 2}">Provide Your <br/>Contact Information</li>
+					ng-show="reporter.shouldRequireLocation() || !reporter.subCategory.shouldDisableForm">Enter a <br/>Location</li>
```

Comment:
@tmccoy529 just waiting for you to roll this back, and then it's good to go, nice work.